internal/runtime/runtime_test.go

```diff
@@ -394,13 +324,8 @@ func TestProcessMessageCBOR(t *testing.T) {
 	runtime.SetTransforms([]appcontext.AppFunction{transform1})
 
 	result := runtime.ProcessMessage(context, envelope)
-	if !assert.Nil(t, result, "result should be null") {
-		t.Fatal()
-	}
-
-	if !assert.True(t, transform1WasCalled, "transform1 should have been called") {
-		t.Fatal()
-	}
+	require.Nil(t, result, "result should be null")
+	assert.True(t, transform1WasCalled, "transform1 should have been called")
```

Comment:
are you not doing require since its the last assertion in the tests?

Reply:
And the fact that that validation failing does impact further testing it there was some. Actually line 327 should be an assert since result isn't use beyond this line.